### ISSUE TITLE
Allow PDF conversion to be cancelled

### ISSUE THREAD

**Issue by pferreir** [indico/indico#2063]
In case people upload the wrong file, etc. I guess we could use Redis to store some piece of information that makes Indico discard the file once it receives it?
To be discussed.

**Comment by ThiefMaster** [comment]
we already discard the pdf file when the "parent" attachment doesn't exist anymore

```python
    if not attachment or attachment.is_deleted or attachment.folder.is_deleted:
        ConversionPlugin.logger.warning('Attachment has been deleted: {}'.format(attachment))
        return jsonify(success=True)
```

**Comment by pferreir** [comment]
Yes, but not in other cases (e.g. the user has given up on it and decided to convert manually to PDF). And we're not hiding the indicator from the interface until we get the file back, IIRC.

**Comment by ThiefMaster** [comment]
Since we use the "after attachment" template hook deleting the attachment should get rid of the indicator, too
